File: client/unity/Assets/Scripts/Assembly-CSharp/APartInfo.cs
public class APartInfo : Part
{
	public sbyte level;

	public sbyte gender;

	public short[] imgID;

	public sbyte[] dx;

	public sbyte[] dy;

	public override void paintAvatar(MyGraphics g, int index, int x, int y, int direct, int arthor)
	{
		if (IDPart == -1)
		{
			return;
		}
		if (IDPart >= 2000)
		{
			ImageIcon imagePart = AvatarData.getImagePart(imgID[index]);
			if (imagePart.count != -1)
			{
				g.drawRegion(imagePart.img, 0f, 0f, imagePart.w, imagePart.h, direct, x + dx[index] * AvMain.hd - ((direct == Base.LEFT) ? (dx[index] * AvMain.hd * 2 + imagePart.w) : 0), y + dy[index] * AvMain.hd, 0);
			}
		}
		else
		{
			ImageInfo imageInfo = AvatarData.listImgInfo[imgID[index]];
			AvatarData.paintPart(g, imageInfo.bigID, imageInfo.x0, imageInfo.y0, imageInfo.w, imageInfo.h, x + dx[index] * AvMain.hd - ((direct == Base.LEFT) ? (dx[index] * AvMain.hd * 2 + imageInfo.w * AvMain.hd) : 0), y + dy[index] * AvMain.hd, direct, 0);
		}
	}
}

File: client/java/src/avt/APartInfo.java
package avt;

import javax.microedition.lcdui.Graphics;

public final class APartInfo extends Part {
   public byte level;
   public byte gender;
   public short[] imgID;
   public byte[] dx;
   public byte[] dy;

   public final void paintAvatar(Graphics var1, int var2, int var3, int var4, int var5) {
      if (super.IDPart != -1) {
         if (super.IDPart >= 2000) {
            ImageIcon var6;
            if ((var6 = AvatarData.getImagePart(this.imgID[var2])).count != -1) {
               var1.drawRegion(var6.img, 0, 0, var6.w, var6.h, var5, var3 + this.dx[var2] * AvMain.hd - (var5 == Base.LEFT ? (this.dx[var2] * AvMain.hd << 1) + var6.w : 0), var4 + this.dy[var2] * AvMain.hd, 0);
               return;
            }
         } else {
            ImageInfo var7 = AvatarData.listImgInfo[this.imgID[var2]];
            AvatarData.drawImgRegion(var1, var7.bigID, var7.x0, var7.y0, var7.w, var7.h, var3 + this.dx[var2] * AvMain.hd - (var5 == Base.LEFT ? (this.dx[var2] * AvMain.hd << 1) + var7.w * AvMain.hd : 0), var4 + this.dy[var2] * AvMain.hd, var5, 0);
         }
      }

   }
}
